acceptance_modelisation/radial_acceptance_map_creator.py

```diff
@@ -16,10 +16,13 @@ def __init__(self,
                  offset_axis: MapAxis,
                  oversample_map: int = 10,
                  exclude_regions: Optional[List[SkyRegion]] = None,
-                 min_observation_per_cos_zenith_bin: int = 3,
+                 cos_zenith_binning_method: str = 'min_livetime',
+                 cos_zenith_binning_parameter_value: int = 3600,
                  initial_cos_zenith_binning: float = 0.01,
+                 max_angular_separation: float = 0.4,
```

Comment:
Same comment as for  get_unique_wobble_pointings function on astropy quantity and default value.
Update of the name to specify it's related to wobble could help code clarity, eg:  max_angular_separation_wobble.

Reply:
Done with your suggested name